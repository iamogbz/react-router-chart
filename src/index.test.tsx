import * as chart from ".";

describe("index", () => {
    it("exports expected values", () => {
        expect(chart).toMatchSnapshot();
    });
});
