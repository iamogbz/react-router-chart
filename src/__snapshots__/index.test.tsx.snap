// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index exports expected values 1`] = `
{
  "$": Symbol($),
  "$$": Symbol($$),
  "Chart": [Function],
}
`;
